src/LIBSVM.jl

```diff
@@ -282,7 +307,7 @@ input matrix `X` for linear kernel.
 function svmtrain(
         X::AbstractMatrix{U}, y::AbstractVector{T} = [];
         svmtype::Type = SVC,
-        kernel::Kernel.KERNEL = Kernel.RadialBasis,
+        kernel = Kernel.RadialBasis,
```

Comment:
Could you update the docstring as well? Just note that the `kernel` argument could be `Base.Callable`.

Reply:
Updated. Let me know if it works for you.